### PR TITLE
Removed the PartnerDomain parameter

### PR DESCRIPTION
**Pull request by gadamyan**
## Description

- Removed the PartnerDomain parameter from the WebView
- added RPM dependency

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ To update the plugin submodule paste the following command in the terminal from
 ## Quick Start
 
 A demo map is included in the plugin for demonstrating how the **WebView** opens at runtime. It is located in the `RpmWebView\Content\Maps` folder.
+The webview will not run properly until the **Subdomain** of your application is set in the project settings. You can find them in **Ready Player Me Studio** website.
+The **Subdomain** should belong to the same application otherwise the webview will fail.
```
Comment (HarrisonHough) [inline README.md:56]: I don't know if this sentence (the subdomain should belong to the same application....) is necessary. I think it was copy and paste from the AppID + subdomain stuff from avatar creator?

File: README.md [hunk 2]
```diff
@@ -52,6 +52,8 @@ To update the plugin submodule paste the following command in the terminal from
 ## Quick Start
 
 A demo map is included in the plugin for demonstrating how the **WebView** opens at runtime. It is located in the `RpmWebView\Content\Maps` folder.
+The webview will not run properly until the **Subdomain** of your application is set in the project settings. You can find them in **Ready Player Me Studio** website.
```
Comment (HarrisonHough) [inline README.md]: maybe add a link to studio here too?

**Review by HarrisonHough — APPROVED** [review]
All good just some small suggestions in the comments.

File: README.md (hunk 1 above)
Comment (gadamyan) [reply README.md:56]: There is an assertion, the application will crash in editor if the subdomain is not set

**Pull request merged**